glean-core/src/metrics/denominator.rs

```diff
@@ -81,15 +90,51 @@ impl DenominatorMetric {
     /// Gets the currently stored value as an integer.
     ///
     /// This doesn't clear the stored value.
-    pub fn test_get_value(&self, glean: &Glean, storage_name: &str) -> Option<i32> {
+    pub fn test_get_value(&self, ping_name: Option<String>) -> Option<i32> {
+        crate::block_on_dispatcher();
+        crate::core::with_glean(|glean| self.get_value(glean, ping_name.as_deref()))
+    }
+
+    #[doc(hidden)]
+    pub fn get_value<'a, S: Into<Option<&'a str>>>(
```

Comment:
Is there a reason `test_get_value` doesn't follow the `_sync` suffix pattern of the other methods? Also, I'm a tad worried about someone seeing a method without a `test_` prefix and deciding to use it in non-test code.

Reply:
Yeah, so this is all .. not perfect right now. I have this pattern across several places right now because of our current test layout.
Let me create a followup bug to address this.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1768832